CRUD.c

```diff
@@ -0,0 +1,186 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+
+typedef struct {
+    int id;
+    char name[50];
+    int age;
+} User;
+
+#define FILE_NAME "users.txt"
+
+
+void initializeFile() {
+    FILE *file = fopen(FILE_NAME, "a");
+    if (file == NULL) {
+        printf("Error creating the file!\n");
+        exit(1);
+    }
+    fclose(file);
+}
+
+
+void createUser() {
+    FILE *file = fopen(FILE_NAME, "a");
+    if (file == NULL) {
+        printf("Error opening the file!\n");
+        return;
+    }
+
+    User user;
+    printf("Enter ID: ");
+    scanf("%d", &user.id);
+    printf("Enter Name: ");
+    scanf(" %[^\n]s", user.name); 
+    printf("Enter Age: ");
+    scanf("%d", &user.age);
+
+    fprintf(file, "%d,%s,%d\n", user.id, user.name, user.age);
+    fclose(file);
+
+    printf("User added successfully!\n");
+}
+
+
+void readUsers() {
+    FILE *file = fopen(FILE_NAME, "r");
```

Comment:
There is no need to open and close the file every time. This can be done only once.